### DISCUSSION TITLE
Get the evaluation of text classification to show accuracy and not just f1

### DISCUSSION THREAD

**Discussion by jmugan** [explosion/spaCy#12687] [Help: Coding & Implementations] [answered]
## How to reproduce the behaviour
Run text classification and tell it to give you accuracy.

It returns precision, recall, F, TOK, TEXCAT (macro F), and ROC AUC. But no accuracy? Is accuracy one of those? I've been googling around and have seen lots of contradictory stuff.

I'm doing binary classification with balanced classes, so I would appreciate the intuitive simplicity of accuracy.

**Comment by shadeMe** [comment, marked as answer]
spaCy doesn't currently provide an accuracy scorer, so you'd need to implement one yourself. Please refer to [this answer](https://github.com/explosion/spaCy/discussions/12682#discussioncomment-6036758) for more information.

**Reply by jmugan** [reply]
Do you know why? Accuracy has its limitations, but it is the first thing somebody would look for. On a funny note, this reminds me of the meme "Do I look like I know what a jpeg is? I just want a picture of a god-dang hot dog" https://www.youtube.com/watch?v=QwucZK1BCj4&ab_channel=freshice211

**Reply by adrianeboyd** [reply]
We're trying to provide one good default scorer that works for a wide range of use cases, and since accuracy is often misleading for unbalanced classes and cases with balanced classes are relatively uncommon, we don't include it by default.
